static/sass/_snapcraft_p-notifications.scss

```diff
@@ -1,20 +1,4 @@
-@mixin snapcraft-p-beta-notification {
-  .p-beta-notification {
-    padding: {
-      bottom: $sp-small;
-      top: $sp-small;
-    }
-
-
-    .p-muted-heading {
-      margin-right: $sp-medium;
-    }
-
-    p {
-      max-width: 100%;
-    }
-  }
-
+@mixin snapcraft-p-notification {
```

Comment:
Remove `p-beta-notification` as we removed it from the site a long time ago.
Updated name of the mixin to reflect it's not just for the beta-notification